### ISSUE TITLE
Shortcut: Paste in Place

### ISSUE THREAD

**Issue by ZachIsAGardner** [aseprite/aseprite#4047]
When copying and pasting something, it would be nice if there was a way to force the paste to be in the original position.

In this gif I'm moving this characters teeth onto a new layer by cutting and pasting them.
![2023_09_18 2](https://github.com/aseprite/aseprite/assets/30502195/3737456f-9d91-47f1-b078-6efb9328000d)

As you can see, the paste works as I'd like in this case if I'm zoomed out, it would be nice if there was a special paste shortcut to force a paste in place. Something like ctrl+shift+v by default.

Thanks!

### Aseprite and System version

* Aseprite version: v1.3-rc6-x64
* System: Windows 11

**Comment by dacap** [comment]
Hi @ZachIsAGardner, seeing your screen record it looks strange, because by default Aseprite tries to paste in the same place where the image was cut, but the position can be different if the visible space is not enough to show the pasted image (e.g. the image in the clipboard is too big to enter in the visible canvas/original position). Could you try the same steps but zooming it out before you paste?

I think there is a bug to fix here, but still not sure what is the specific position/size of selection that you are trying to cut/paste which might be making Aseprite behaving in a specific unexpected way.

**Comment by ZachIsAGardner** [comment]
You're right, the selection I'm making is bigger than what is visible on screen. So I think it's behaving as expected. In my screen recording I do zoom out to paste the 2nd time and it pastes in place correctly.

I'm more-so proposing a feature, not reporting a bug. In my use case I'm just trying to move some pixels from one layer to another. In my screen recording I have a row of 48x48 sprites that is 6 tiles wide. I highlight my characters teeth for each of those 48x48 sprites and I cut/paste them onto a new layer. This only works as I'm intending if I'm zoomed out, but I'm zoomed in at like 800% in order to properly highlight the right pixels.

I understand why the paste works the way it does, what I'm wondering is if we can add a new shortcut that always pastes in the original spot no matter what. This would eliminate the step where I have to zoom out after I've made my selections.

So there would be two paste commands. The normal paste (CTRL+V) that factors in screen space and the new "paste in place" (CTRL+SHIFT+V) that doesn't care about screen space for use cases like mine.

I've seen this in other programs like Adobe Photoshop.
![Screenshot 2023-09-19 164941](https://github.com/aseprite/aseprite/assets/30502195/9ad78ca4-0b00-4666-933e-7fc0501c6224)

**Comment by numerfolt** [comment]
I probably have the same bug?

Sometimes when copy pasting a selection, it will not paste it in place.
Most of the times it does, but sometimes it doesnt.

In the appended video, I try to replicate it for 1 minute, but it always behaves correctly. 
At 1:20 I try another selection and it happens immediately.

https://github.com/user-attachments/assets/2167d26d-66d3-411b-91f4-0b6e4fccb197
